imagebuildah/build.go

```diff
@@ -329,6 +343,12 @@ func (b *Executor) Copy(excludes []string, copies ...imagebuilder.Copy) error {
 		for _, src := range copy.Src {
 			if strings.HasPrefix(src, "http://") || strings.HasPrefix(src, "https://") {
 				sources = append(sources, src)
+			} else if len(copy.From) > 0 {
+				if other, ok := b.named[copy.From]; ok {
```

Comment:
We'll need to check that the stage number for `other` is less than the one for `b`.

Reply:
Done.